docs/achtergronddocumentatie/artefacten/9853.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page-with-side-nav
+title: C9853 - Klant appt een vraag naar de gemeente die beantwoord kan worden door KCC.
+date: 24-01-2024
+---
+
+# C9853 - Klant appt een vraag naar de gemeente die beantwoord kan worden door KCC.
+
+Deze casus is een variant op [C1199](./1199.md).
+
+Aangenomen wordt dat het KCC het app-verkeer afhandelt. 
+ 
+Aangenomen wordt dat er software gebruikt wordt om het app-verkeer met een persoon geautomatiseerd te kunnen verwerken tot een eenheid van vastlegging.
+
+***Het grote vraagstuk binnen deze case is wat de eenheid van vastlegging omvat. Wat is een afgerond "app-gesprek". Hier zal zowel het tijds-aspect als de (afgeronde) inhoud een rol spelen.***
+
+## 1. Informatieobject wordt geregistreerd
```

Comment:
Optioneel? Verslag kan misschien ook in de klantcontactinhoud geïnjecteerd worden?